website/static/js/pages/files-page.js

```diff
@@ -14,10 +15,19 @@ $(document).ready(function(){
     $.ajax({
       url: nodeApiUrl + 'files/grid/'
     }).done(function(data) {
+        function register() {
+            if(node.isRegistered){
```

Comment:
This could be simplified to `return !node.isRegistered`. If you have access to the data in the Fangorn setup, then `allowMove: !node.isRegistered` on those two lines will probably do what you need.